crates/levm/src/utils.rs

```diff
@@ -0,0 +1,9 @@
+use ethereum_types::U256;
+
+pub fn u256_to_i128(value: U256) -> i128 {
```

Comment:
Maybe add a short comment explaining that this returns the signed integer from its 2's complement representation.

Reply:
Sure, will do!

Reply:
Casting to `i128` was incorrect in these cases so I removed that function.